reana_server/rest/workflows.py

```diff
@@ -869,12 +877,12 @@ def upload_file(workflow_id_or_name):  # noqa
           description: Required. Analysis UUID or name.
           required: true
           type: string
-        - name: file_content
-          in: formData
-          description: >-
-            Required. File to be transferred to the workflow workspace.
+        - name: file
+          in: body
+          description: Required. File to add to the workspace.
```

Comment:
Shall we update `reana-commons` with the new r-server and r-w-controller OpenAPI specs as well?

Reply:
Created https://github.com/reanahub/reana-commons/pull/141